storage/google/cloud/storage/client.py

```diff
@@ -394,6 +404,82 @@ def download_blob_to_file(self, blob_or_uri, file_obj, start=None, end=None):
 
             blob_or_uri.download_to_file(file_obj, client=self, start=start, end=end)
 
+    def list_blobs(
+        self,
+        bucket_or_name,
+        max_results=None,
+        page_token=None,
+        prefix=None,
+        delimiter=None,
+        versions=None,
+        projection="noAcl",
+        fields=None,
+    ):
+        """Return an iterator used to find blobs in the bucket.
+
+        If :attr:`user_project` is set, bills the API request to that project.
+
+        bucket_or_name (Union[ \
+            :class:`~google.cloud.storage.bucket.Bucket`, \
+             str, \
+        ]):
+            The bucket resource to pass or name to create.
+
+        :type max_results: int
```

Comment:
Use Google-style docstrings. https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google

Reply:
Okey, it's done